src/aggregation/models/validations.ts

```diff
@@ -0,0 +1,73 @@
+import { VALIDATIONS, type AggregationLayerMetadata } from '@map-colonies/mc-model-types';
+import type { MultiPolygon, Polygon } from 'geojson';
+import { z, type ZodType } from 'zod';
+
+export const aggregationMetadataSchema: ZodType<AggregationLayerMetadata> = z.object(
```

Comment:
move to mc-models in order to support reusing

Reply:
done